src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/AccessToken/OidcTokenHandlerFactory.php

```diff
@@ -60,6 +53,38 @@ public function addConfiguration(NodeBuilder $node): void
         $node
             ->arrayNode($this->getKey())
                 ->fixXmlConfig($this->getKey())
+                ->validate()
+                    ->ifTrue(static function ($v) {return !isset($v['algorithm']) && !isset($v['algorithms']); })
```

Comment:
let's use short closures everywhere possible (please update the other occurences):
```suggestion
                    ->ifTrue(static function ($v) => !isset($v['algorithm']) && !isset($v['algorithms']))
```